### PR TITLE
Add aria2 (aria2c) to Windows

### LINKED ISSUE #3670: Add aria2 on windows
### Tool name

aria2

### Tool license

GPL 2

### Add or update?

- [X] Add
- [ ] Update

### Desired version

1.35.0

### Approximate size

2MB

### Brief description of tool

aria2 is already available on linux and on macOS

aria2 - The ultra fast download utility

aria2 is a utility for downloading files. The supported protocols are HTTP(S), FTP, SFTP, BitTorrent, and Metalink. aria2 can download a file from multiple sources/protocols and tries to utilize your maximum download bandwidth. It supports downloading a file from HTTP(S)/FTP/SFTP and BitTorrent at the same time, while the data downloaded from HTTP(S)/FTP/SFTP is uploaded to the BitTorrent swarm. Using Metalink's chunk checksums, aria2 automatically validates chunks of data while downloading a file like BitTorrent.

### URL for tool's homepage

https://github.com/aria2/aria2

### Provide a basic test case to validate the tool's functionality.

```bash
aria2c --version
```

### Virtual environments affected

- [ ] Ubuntu 16.04
- [ ] Ubuntu 18.04
- [ ] Ubuntu 20.04
- [ ] macOS 10.15
- [ ] macOS 11
- [X] Windows Server 2016
- [X] Windows Server 2019

### Can this tool be installed during the build?

```bash
choco install
```

### Tool installation time in runtime

1s

### Are you willing to submit a PR?

Yes

### PR DESCRIPTION
**Pull request by jmarrec**
# Description

Fix #3670 - Add aria2 (aria2c) to Windows

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/Windows2016-Readme.md [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
 ### Tools
 - 7zip 19.00
+- aria2 1.35.0
```
Comment (miketimofeev) [inline images/win/Windows2016-Readme.md:43]: There is no need to update the Readme manually, need to update Software Report Generator instead
https://github.com/actions/virtual-environments/tree/main/images/win/scripts/SoftwareReport

Comment (jmarrec) [reply images/win/Windows2016-Readme.md]: Done in 4543939c7c0c27865a0f164306aee086320565ae

File: images/win/scripts/Installers/Install-Aria2.ps1 [hunk 2]
```diff
@@ -0,0 +1,8 @@
+################################################################################
+##  File:  Install-Aria2.ps1
+##  Desc:  Install Aria2
+################################################################################
+
+Choco-Install -PackageName Aria2
```
Comment (miketimofeev) [inline images/win/scripts/Installers/Install-Aria2.ps1:6]: nitpick:
```suggestion
Choco-Install -PackageName aria2
```
image

File: images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1 [hunk 3]
```diff
@@ -1,3 +1,9 @@
+function Get-Aria2Version {
+    ((cmd /c "aria2c -- version 2>&1") | Out-String) -match "aria2 version (?<version>\d+\.\d+\.\d+)" | Out-Null
```
Comment (miketimofeev) [inline images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1:2]:
```suggestion
    (aria2c -v | Out-String) -match "(?<version>\d+\.{1,}\d+)" | Out-Null
```

**Review by Dg600u — APPROVED** [review]

File: images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1 (hunk 3 above)
Comment (jmarrec) [reply images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1:2]: Unless powershell has a very unusual regex syntax, I think this suggested regex is broken?

Comment (miketimofeev) [reply images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1:2]: @jmarrec sorry, my fault, there should be brackets, fixed the suggestion

**Comment by AlenaSviridenko** [comment]
@jmarrec, please, pull the latest main branch and I will run test CI to validate your changes.

**Comment by AlenaSviridenko** [comment]
Hi @jmarrec,
gentle ping, is this change still needed?

**Comment by AlenaSviridenko** [comment]
I am going to close this PR and the related issue. Please, feel free to ping us for reopening if it is still actual. Thanks!

**Pull request closed without merging**